src/app/leverage/page.tsx

```diff
@@ -31,7 +34,17 @@ export default function Page() {
     onOpen: onOpenSelectBaseToken,
     onClose: onCloseSelectBaseToken,
   } = useDisclosure()
-  const { baseToken, baseTokens, onSelectBaseToken } = useLeverageToken()
+  const { baseToken, baseTokens, indexToken, onSelectBaseToken } =
+    useLeverageToken()
+  const [currentTab, setCurrentTab] = useState<ChartTab>('indexcoop-chart')
+
+  useEffect(() => {
+    document.body.classList.add('bg-ic-dark')
+    return () => {
+      document.body.classList.remove('bg-ic-dark')
+    }
+  })
```

Comment:
Design request to ensure the background is dark when user scrolls/pulls down on mobile, so background needs to be applied to `body`. The root layout is 'light', so we would have needed to create two separate route groups for dark and light, and split the code by route group.

Not ideal, but I think this is preferred